### PR TITLE
Add info about notOptimized parameter to readme

### PR DESCRIPTION
**Pull request by tvinzz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,16 @@ If you want to help with filters translations, you can join us on Crowdin: https
 
 Please learn more about translating our products: https://kb.adguard.com/general/adguard-translations
 
+## <a id="templates"></a> Filters templates
+`@include` directive allows to include contents of specified file into the filter.
+
+If you want to add a filter and keep all their rules during optimization, use the `/notOptimized` parameter.
```
Comment (slavaleleka) [inline README.md:143]: наверное не `their` а `its`

File: README.md [hunk 2]
```diff
@@ -137,6 +137,16 @@ If you want to help with filters translations, you can join us on Crowdin: https
 
 Please learn more about translating our products: https://kb.adguard.com/general/adguard-translations
 
+## <a id="templates"></a> Filters templates
+`@include` directive allows to include contents of specified file into the filter.
```
Comment (slavaleleka) [inline README.md:141]: `to include the content`

File: README.md [hunk 3]
```diff
@@ -137,6 +137,16 @@ If you want to help with filters translations, you can join us on Crowdin: https
...
 Please learn more about translating our products: https://kb.adguard.com/general/adguard-translations
 
+## <a id="templates"></a> Filters templates
+`@include` directive allows to include the content of specified file into the filter.
+
+If you want to add a filter and keep all its rules during optimization, use the `/notOptimized` parameter.
+
+Example:
+```
+@include "https://example.org/test-filter.txt" /notOptimized
```
Comment (Alex-302) [inline README.md:147]: Can we use this already?

Comment (tvinzz) [reply README.md:147]: @Alex-302, we use this parameter for few years already.
for example: https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/filter_2_English/template.txt#L68

**Review by Alex-302 — APPROVED** [review]

**Review by slavaleleka — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**